sdk/src/store.rs

```diff
@@ -2311,8 +2342,8 @@ impl Store {
                 )
                 .provenance
                 {
-                    // return an error with the url that should be read
```

Comment:
I don't think we need both a public **load_cai_from_memory** and public **load_from_memory**.  We have two paths that do the same thing.  I think we can remove public from **load_cai_from_memory** .  I think only the "load_from" functions should be public.

Reply:
Made the change - will this require a `(MINOR)` tag in the PR title?